<?xml version="1.0"?>

<launch>
	<node respawn="true" pkg="rovitis_phidgets" type="phidgets_analog_4_output" name="phidgets_analog_4_output" output="log">
		<param name="dev_serial_number" type="int" value="275374" />
		<param name="dev_serial_label" type="string" value="phidgets_analog_4_output" />
		<param name="frame" type="string" value="/phidget_ao" />
		<param name="wait_for_dev_delay" type="int" value="10000" />
		<param name="publish_frequency" type="int" value="5" />
		<param name="topic_name" type="string" value="/phidgets_analog_4_output" />
		<param name="set_service_name" type="string" value="phidgets_analog_4_output_set" />
		<param name="enable_ch1" type="int" value="1" />
		<param name="enable_ch2" type="int" value="1" />
		<param name="enable_ch3" type="int" value="1" />
		<param name="enable_ch4" type="int" value="1" />		
	</node>
</launch>
